docs/compliance/index.md

```diff
@@ -4,10 +4,27 @@ keywords: standards, compliance, security
 title: Docker standards and compliance
 ---
 
-We want to ensure that Docker Enterprise Edition can be used in a manner that meets the requirements of various security and compliance standards. You can use this documentation in order to validate your Docker Enterprise Edition deployment against applicable security controls and configuration baselines. The documents that we've highlighted thus far are as follows:
+Docker Enterprise Edition can be configured and used in a manner that can help your organization satisfy the requirements of various security and compliance standards. You can use this documentation in order to validate your Docker Enterprise Edition deployment against applicable security controls and configuration baselines. The catalogs, frameworks, publications and benchmarks that we've highlighted thus far are as follows:
```

Comment:
This is wordy and could maybe be simplified.

Docker Enterprise Edition can be configured and used in accordance with various security and compliance standards. Use this document to validate your deployment against applicable....

Also you need a comma after "publications". We use Oxford / serial commas. :)

Reply:
Awesome! Will update